.github/release-drafter.yml

```diff
@@ -0,0 +1,33 @@
+name-template: 'Wasm-Filters v$NEXT_PATCH_VERSION'
+tag-template: 'v$NEXT_PATCH_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'kind/feature'
+      - 'kind/enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'kind/fix'
+      - 'kind/bugfix'
+      - 'kind/bug'
+  - title: '🧰 Maintenance'
+    labels:
+      - 'kind/chore'
+      - 'area/ci'
+      - 'area/tests'
```

Comment:
It is worth noting that some of the labels mentioned here are not present for this repository yet.
Since they are standard for almost all of the Layer5 projects, the changes here are valid and the labels for this repository should be updated accordingly. //@leecalcote